Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -875,6 +875,24 @@ theorem isCoprime_span_singleton_iff (x y : R) :
   · rintro ⟨a, _, ⟨b, rfl⟩, e⟩; exact ⟨a, b, mul_comm b y ▸ e⟩
   · rintro ⟨a, b, e⟩; exact ⟨a, _, ⟨b, rfl⟩, mul_comm y b ▸ e⟩
 
+theorem isCoprime_biInf {J : ι → Ideal R} {s : Finset ι}
+    (hf : ∀ j ∈ s, IsCoprime I (J j)) : IsCoprime I (⨅ j ∈ s, J j) := by
+  classical
+  simp_rw [isCoprime_iff_add] at *
+  revert hf
+  induction s using Finset.induction with
```

Comment:
I think `induction` does this `revert` automatically?

Reply:
Good catch!